### PR TITLE
[4.3] Convert missing files to PSR-12 code style

### PR DESCRIPTION
**Pull request by heelc29**
Pull Request for Issue #38754 and #38786 .

### Summary of Changes

Note that not all files were converted to the PSR-12 codestyle during PR #37686 eg:
- administrator/components/com_actionlogs/src/View/Action`logs/`HtmlView.php
- administrator/components/com_admin/src/Service/HTML/`Configuration.php`
- administrator/components/com_cache/tmpl/`cache/`default.php
- administrator/components/com_media/tmpl/`media/`default.php
- api/components/com_`media/`src/Model/MediaModel.php
- layouts/joomla/form/field/`media/`accessiblemedia.php
- libraries/src/`Cache/`Cache.php
- libraries/src/Component/Router/RouterView`Configuration.php`
- plugins/fields/`media/`media.php
- plugins/privacy/action`logs/`actionlogs.php
- plugins/system/`cache/`src/Extension/Cache.php
- plugins/webservices/`media/`media.php

These files are therefore not checked by Drone now. This is due to an overly open exclude list (marked in list above).

### Testing Instructions

- Drone works
- Code Review
- Install Joomla

### Actual result BEFORE applying this Pull Request

2648 files are checked by `phpcs`

### Expected result AFTER applying this Pull Request

2743 files are checked by `phpcs`

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

cc @HLeithner

### CODE REVIEW

**Comment by richard67** [comment]
@heelc29 Regarding the testing instructions: "Install Joomla" is not necessary. That is included already in the "Drone works". The API and system tests include making a new installation on diverse environments, so if these pass, installation works. But it could make sense to let people run the PHPCS tests locally in their environment with `./libraries/vendor/bin/phpcs --extensions=php -p --standard=ruleset.xml .` and check that this works.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 0de1eff0ae04f780747d5061e08708390657b020<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39197">issues.joomla.org/tracker/joomla-cms/39197</a>.</sub>

**Comment by heelc29** [comment]
@richard67 

> Regarding the testing instructions: "Install Joomla" is not necessary

Just copied from PR #37686 - but only a part of it 😅

> But it could make sense to let people run the PHPCS tests locally in their environment

This is not always possible because I have problems with the EOL in my **win** environment:
image

**Comment by brianteeman** [comment]
@heelc29 there is usually a setting in your git client that will resolve that

**Comment by richard67** [comment]
@heelc29 Why have you updated your branch without any need? It resets the counter for human tests and I manually have to add the test result again in the issue tracker. That causes unnecessary work and may cause the one or other PR with 2 good test results being overlooked.

**Comment by heelc29** [comment]
> @heelc29 there is usually a setting in your git client that will resolve that

@brianteeman I use GitHub Desktop with default settings ... mmmh so I can't be sure that everyone else has changed it at their git client

**Comment by heelc29** [comment]
> @heelc29 Why have you updated your branch without any need? It resets the counter for human tests and I manually have to add the test result again in the issue tracker. That causes unnecessary work and may cause the one or other PR with 2 good test results being overlooked.

@richard67 I thought to make sure there are no conflicts or new code style errors from the changed ruleset.

**Comment by brianteeman** [comment]
> . mmmh so I can't be sure that everyone else has changed it at their git client

Thats irrelevant. Just follow https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

File: build/psr12/ruleset.xml [hunk 1]
```diff
@@ -245,6 +239,7 @@
         <exclude-pattern type="relative">plugins/content/pagebreak/pagebreak\.php</exclude-pattern>
         <exclude-pattern type="relative">plugins/editors/none/none\.php</exclude-pattern>
         <exclude-pattern type="relative">plugins/user/joomla/joomla\.php</exclude-pattern>
+        <exclude-pattern type="relative">tests/</exclude-pattern>
```
Comment (laoneo) [inline build/psr12/ruleset.xml:242]: Why do you exclude here tests?

Comment (heelc29) [reply build/psr12/ruleset.xml:242]: I just copied the root folder's ruleset, which was adjusted with PR #38161.

**Comment by heelc29** [comment]
Conflicts resolved in:
- administrator/components/com_actionlogs/tmpl/actionlogs/default.php
- libraries/src/Cache/Cache.php
- libraries/src/Cache/CacheController.php
- libraries/src/Cache/CacheStorage.php
- libraries/src/Cache/Controller/CallbackController.php
- libraries/src/Cache/Storage/FileStorage.php
- libraries/src/Cache/Storage/MemcachedStorage.php
- libraries/src/Cache/Storage/RedisStorage.php
- plugins/privacy/actionlogs/actionlogs.php
- plugins/system/actionlogs/actionlogs.php

File: administrator/components/com_cache/tmpl/cache/default.php [hunk 2]
```diff
@@ -43,51 +44,52 @@
...
-                            </th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <?php $i = 0; ?>
-                        <?php foreach ($this->data as $folder => $item) : ?>
-                            <tr class="row<?php echo $i % 2; ?>">
-                                <td>
-                                    <?php echo HTMLHelper::_('grid.id', $i, $item->group, false, 'cid', 'cb', $item->group); ?>
+                    <table class="table">
```
Comment (HLeithner) [inline administrator/components/com_cache/tmpl/cache/default.php:47]: can you revert the changing to the template file, we don't have a clear psr policy for mixed html/php files

**Review by HLeithner — CHANGES_REQUESTED** [review]
sorry that this took so long, can you change the target branch to 4.3 pleasE?

**Comment by heelc29** [comment]
> sorry that this took so long, can you change the target branch to 4.3 pleasE?

@HLeithner Sure, but I will wait for the next upmerge, there are many conflicts at the moment:
_[4.3] Change application input access to getInput_ #39029 vs. _Changes the code base to short array syntax_ #39616
And a tricky one too in `libraries/namespacemap.php` which I don't want to resolve. Take care here @obuisard ;)
_[4.2] Fix library autoloading_ #39348 vs. _Add namespace support to templates_ #39011

File: administrator/components/com_cache/tmpl/cache/default.php (hunk 2 above)
Comment (heelc29) [reply administrator/components/com_cache/tmpl/cache/default.php:47]: @HLeithner If I use the current file of the 4.2-dev branch I will get following errors:
image

If I should revert the changes, I have to add this file to ignore list to get drone phpcs happy (again). Should I do this?

Comment (HLeithner) [reply administrator/components/com_cache/tmpl/cache/default.php:47]: in this case keep it

Comment (heelc29) [reply administrator/components/com_cache/tmpl/cache/default.php:47]: Ok

**Comment by heelc29** [comment]
> sorry that this took so long, can you change the target branch to 4.3 pleasE?

@HLeithner done

**Review by HLeithner — APPROVED** [review]
Thanks

**Pull request merged**